Formula/otf2bdf.rb

```diff
@@ -1,7 +1,7 @@
 class Otf2bdf < Formula
   desc "OpenType to BDF font converter"
-  homepage "http://sofia.nmsu.edu/~mleisher/Software/otf2bdf/"
-  url "http://sofia.nmsu.edu/~mleisher/Software/otf2bdf/otf2bdf-3.1.tbz2"
+  homepage "https://web.archive.org/web/20220629172211/http://sofia.nmsu.edu/~mleisher/Software/otf2bdf/"
```

Comment:
Do we routinely do these for deprecations? Given it has `cellar: :any` IMO we can just leave it as-is and merge despite failing CI.